chef/cookbooks/crowbar/templates/default/crowbar-pre-upgrade.sh.erb

```diff
@@ -4,6 +4,7 @@
 
 LOGFILE=/var/log/crowbar/node-upgrade.log
 mkdir -p "`dirname "$LOGFILE"`"
+chmod 600 $LOGFILE
```

Comment:
It probably does exist (from some previous script) but you are right I should move this to later part